app/javascript/crayons/MultiInput/MultiInput.jsx

```diff
@@ -0,0 +1,117 @@
+import { h, Fragment } from 'preact';
+import { useRef, useState } from 'preact/hooks';
+
+const KEYS = {
+  ENTER: 'Enter',
+  COMMA: ',',
+  SPACE: ' ',
+};
+// TODO: think about how this may change based on
```

Comment:
I've added some specific todo notes in this file that I will tackle in a subsequent PR.